client/blocks/signup-form/index.jsx

```diff
@@ -896,13 +892,48 @@ class SignupForm extends Component {
 						) }
 					</LoggedOutForm>
 
-					<LoggedOutFormLinkItem href={ logInUrl }>
+					<LoggedOutFormLinkItem href={ this.getloginUrl() }>
 						{ this.props.translate( 'Log in with an existing WordPress.com account' ) }
 					</LoggedOutFormLinkItem>
 				</div>
 			);
 		}
 
+		/*
+			AB Test: passwordlessSignup
+
+			`<PasswordlessSignupForm />` is for the `onboarding` flow.
+
+			We are testing whether a passwordless account creation and login improves signup rate in the `onboarding` flow
+		*/
+		if (
+			this.props.flowName === 'onboarding' &&
+			'passwordless' === abtest( 'passwordlessSignup' )
+		) {
+			return (
+				<div className={ classNames( 'signup-form', this.props.className ) }>
+					{ this.getNotice() }
+					<PasswordlessSignupForm
+						stepName={ this.props.stepName }
+						flowName={ this.props.flowName }
+						goToNextStep={ this.props.goToNextStep }
+						userCreationComplete={ this.userCreationComplete() }
+						renderTerms={ this.termsOfServiceLink() }
+						getloginUrl={ this.getloginUrl() }
```

Comment:
This one also errors out for me in the console if I enter an existing email address (stuck in the loading state), because `<PasswordlessSignupForm>` attempts to call it.

```suggestion
						getloginUrl={ this.getloginUrl }
```